api/v1alpha1/jindoruntime_types.go

```diff
@@ -153,6 +153,10 @@ type JindoRuntimeSpec struct {
 
 	// +optional
 	LogConfig map[string]string `json:"logConfig,omitempty"`
+
+	// If enable hostnetwork and default true
+	// +optional
+	DisableHostNetWork bool `json:"clusternetwork,omitempty"`
```

Comment:
I suggest to use the name  `disableHostNetwork`.

Reply:
ok what about
NetworkMode string `json:"networkmode,omitempty"`

 if empty and default value is Host

Reply:
I prefer `json:"networkMode,omitempty"`. Others are fine to me.